README.md

```diff
@@ -166,6 +166,13 @@ To enable instrumentation libraries manually, see [Manual library instrumentatio
 Documentation on how to manually instrument a Ruby application is available in the 
 [OpenTelemetry official documentation](https://opentelemetry.io/docs/instrumentation/ruby/manual/).
 
+To extend the instrumentation with the OpenTelemetry Instrumentation for Ruby,
+you have to use a compatible API version.
+
+The Splunk Distribution of OpenTelemetry Ruby version 0.1.0 is compatible with:
+
+- OpenTelemetry API version 1.0.0
```

Comment:
Shouldn't we also list supported versions of SDK and instrumentations (`opentelemetry-instrumentation-base`)?

Reply:
I think at least giving the version of the SDK would be good enough

Reply:
Added the SDK.